.gitignore

```diff
@@ -40,3 +40,7 @@ cwthbntv.nci.txt
 *.iml
 real_time/surface_src/java_src/Alice/test.mp4
 t30euete.f4z.txt
+bytes from arduino.txt
+other.txt
```

Comment:
Other.txt is a debug file that shouldn't be in rbsm_console.  We will remove it, and likewise it should be removed here.